cli/tests/integration/test_tests.rs

```diff
@@ -50,7 +50,7 @@ itest!(quiet {
 });
 
 itest!(fail_fast {
-  args: "test --fail-fast test/fail_fast.ts",
+  args: "test test/fail_fast.ts --fail-fast",
```

Comment:
@caspervonb If we keep it 
```ts
test --fail-fast test/fail_fast.ts
```
it will throw this error fail-fast should be a number

Reply:
Yes, and that is how tests are supposed to work!
It throws because the behaviour has been changed in a breaking manner.

Reply:
@caspervonb  With these changes if we run 
```ts
./target/debug/deno test fetcher.ts --fail-fast 3
```
it will throw 
```ts
error: Cannot resolve module "file:///home/getspooky/Documents/open-source/deno/3".
```